packages/cli/test/utils/crucible/kurtosis/test/deneb.yml

```diff
@@ -0,0 +1,35 @@
+participants:
```

Comment:
We have only 2 nodes in the `deneb.test.ts`. Let's try to mimic the same environment here and have only two participants.